### PR TITLE
If input checkbox of selected item is toggled, toggle all selected items

### PR DESCRIPTION
**Pull request by alukach**
selection-update

I think that this should improve upon the 'natural' feel of the layerlist interaction.

Going to need some testing love before merged.

### CODE REVIEW

File: src/js/layerlist/LayerList.js [hunk 1]
```diff
@@ -176,10 +176,30 @@ L.Dropchop.LayerList = L.Control.extend({
     */
     _handleLayerChange: function(obj, e){
         var inputEl = e.target;
+        var selected = obj.domElement.getElementsByClassName('layer-name')[0].classList.contains('selected')
         if (inputEl.checked) {
-            this._map.addLayer(obj.layer);
+            // If changed element in selection, alter entire selection
+            if (selected) {
+                this.selection.list.map(function(obj){
```
Comment (alukach) [inline src/js/layerlist/LayerList.js]: I'm starting to think that basic `for` loops are better than `.map`.  Faster, possibly more intuitive to reader, no need for `.bind`.

**Comment by mapsam** [comment]
Good idea!

**Comment by alukach** [comment]
Fyi, still have some more tests coming down the pipe, let's hold off on merging.

**Comment by alukach** [comment]
Alright, this is good to go!  Added some tests for the layerlist.

**Comment by mapsam** [comment]
Looks great on my end!

**Pull request merged**